src/messaging/ExchangeContext.h

```diff
@@ -199,6 +182,9 @@ class DLL_EXPORT ExchangeContext : public ReliableMessageContext,
     void Free();
     void Reset();
 
+    bool IsResponseExpected() const;
+    void SetResponseExpected(bool inResponseExpected);
```

Comment:
These still need to be documented, especially their interaction with sendflags.

Reply:
Acked